### PR TITLE
Gl 2 front end setup

### PR DESCRIPTION
**Pull request by daniyazafar**
Merge initial setup of partials to dev branch

### CODE REVIEW

**Review by dvyScript — COMMENTED** [review]
looks good

File: src/styles/partials/_mixins.scss
Comment (Muudy12) [inline src/styles/partials/_mixins.scss:1]: Can you move the font faces to a _typography.scss file.

File: src/styles/partials/_variables.scss
Comment (Muudy12) [inline src/styles/partials/_variables.scss:1]: the media queries can be in the _media.scss file. Let's keep the breakpoint variables in the _varables.scss

**Review by Muudy12 — CHANGES_REQUESTED** [review]
Still need some changes before approval.

File: src/styles/partials/_mixins.scss
Comment (daniyazafar) [reply src/styles/partials/_mixins.scss:1]: Done.

File: src/styles/partials/_variables.scss
Comment (daniyazafar) [reply src/styles/partials/_variables.scss:1]: Done

**Comment by Sophia-G-Chan** [comment]
If we use the @ @Muudy12 @daniyazafar do we get email/notifications?

**Review by Muudy12 — APPROVED** [review]

**Pull request merged**
